main.go

```diff
@@ -54,6 +54,9 @@ var DefaultEmailBodyTemplate string
 var configPath = flag.String("config", "rageshake.yaml", "The path to the config file. For more information, see the config file in this repository.")
 var bindAddr = flag.String("listen", ":9110", "The port to listen on.")
 
+const DefaultErrorReason string = "app or user text rejected"
+const DefaultErrorCode string = "RS_UNKNOWN"
```

Comment:
these things can be private. Golang is awful, so the way you define a private symbol is by starting with a lowercase letter:

```suggestion
const defaultErrorReason string = "app or user text rejected"
const defaultErrorCode string = "RS_UNKNOWN"
```